### PR TITLE
fix(ffxiv): improve extended help

### PR DESCRIPTION
**Pull request by favna**

### CODE REVIEW

File: src/languages/es-ES/commands/gameIntegration.json [hunk 1]
```diff
@@ -144,10 +144,12 @@
...
 		"examples": [
+			"item poción",
 			"character Laytlan Ardevon",
-			"Laytlan Ardevon",
-			"poción"
-		]
+			"character Saewynn Frobisher --server=Omega",
+			"Laytlan Ardevon"
+		],
+		"reminder": "When searching for a character you can specify the server the character is playing on by including `--server=ServerName` in your message."
```
Comment (kyranet) [inline src/languages/es-ES/commands/gameIntegration.json]:
```suggestion
		"reminder": "Cuando buscas por un personaje, puedes especificar el servidor en el que juega incluyendo `--server=NombreDelServidor` en tu mensaje."
```

File: src/languages/es-ES/commands/gameIntegration.json [hunk 2]
```diff
@@ -144,10 +144,12 @@
 			]
 		],
 		"examples": [
+			"item poción",
```
Comment (kyranet) [inline src/languages/es-ES/commands/gameIntegration.json]: Does FFXIV understand Spanish names, tho?

**Review by kyranet — CHANGES_REQUESTED** [review]

File: src/languages/es-ES/commands/gameIntegration.json (hunk 2 above)
Comment (favna) [reply src/languages/es-ES/commands/gameIntegration.json]: You're the one who added this lmfao

Comment (favna) [inline src/languages/es-ES/commands/gameIntegration.json]:
```suggestion
			"item potion",
```

**Review by kyranet — APPROVED** [review]

**Pull request merged**